doc/api/test.md

```diff
@@ -2814,6 +2814,11 @@ are defined, while others are emitted in the order that the tests execute.
         numbers and the number of times they were covered.
         * `line` {number} The line number.
         * `count` {number} The number of times the line was covered.
+    * `thresholds` {Object} An object containing whether or not the coverage for
```

Comment:
Just to clarify, this is undefined if --test-coverage-branches or lines/functions options are sent, right?

Reply:
No, each will be marked as `true`, indicating a pass.